File: 0263_Ugly Number/UglyNumber.java
class Solution {
    public boolean isUgly(int num) {
        if(num <=0) return false;
        
        int[] prime = new int[]{2,3,5};
        
        for(int i=0;i<prime.length;i++){
            while(num % prime[i] == 0){
                num /= prime[i];
            }
        }
        
        return num == 1;
    }
}

File: 0263_Ugly Number/UglyNumber.cs
class UglyNumber
{
    public bool IsUgly(int num)
    {
        if (num <= 0) return false;
        int[] sequence = new int [] { 2, 3, 5 };
        for(int i = 0; i < sequence.Length; i++)
        {
            int v = sequence[i];
            while(num % v == 0)
            {
                num /= v;
            }
        }

        return num == 1;
    }
}
